### PR TITLE
Performance best practices add apcu info

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

Add configuration details for APCU. For some reason the APCU details added in PR https://github.com/magento/devdocs/pull/2115 were lost at some point, so re-adding them

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/performance-best-practices/software.html
- https://devdocs.magento.com/guides/v2.2/performance-best-practices/deployment-flow.html

whatsnew
Added instructions for configuring the  PHP APCu extension to the "Software recommendations" and "Deployment flow" topics in the _Performance Best Practices Guide_.

### CODE REVIEW

File: src/guides/v2.2/performance-best-practices/software.md [hunk 1]
```diff
@@ -95,6 +95,18 @@ opcache.memory_consumption=64
 opcache.max_accelerated_files=60000
 ```
 
+#### APCU
+
+We recommend enabling the [PHP APCu extension](https://getcomposer.org/doc/articles/autoloader-optimization.md#optimization-level-2-b-apcu-cache) and [configuring `composer` to support it]({{ page.baseurl }}/performance-best-practices/deployment-flow.html#preprocess-dependency-injection-instructions) to optimize for maximum performance. This extension caches file locations for opened files, which increases performance for Magento server calls including pages, ajax calls, and endpoints.
```
Comment (keharper) [inline src/guides/v2.2/performance-best-practices/software.md]:
```suggestion
We recommend enabling the [PHP APCu extension](https://getcomposer.org/doc/articles/autoloader-optimization.md#optimization-level-2-b-apcu-cache) and [configuring `composer` to support it]({{ page.baseurl }}/performance-best-practices/deployment-flow.html#preprocess-dependency-injection-instructions) to optimize for maximum performance. This extension caches file locations for opened files, which increases performance for Magento server calls including pages, Ajax calls, and endpoints.
```

File: src/guides/v2.2/performance-best-practices/software.md [hunk 2]
```diff
@@ -95,6 +95,18 @@ opcache.memory_consumption=64
...
+
+Edit your `apcu.ini` file to include the following:
+
+```text
+extension=apcu.so
+[apcu]
+acp.enabled = 1
+```
+
```
Comment (keharper) [inline src/guides/v2.2/performance-best-practices/software.md]: Does it matter where these lines are placed?

File: src/guides/v2.3/performance-best-practices/software.md [hunk 3]
```diff
@@ -93,6 +93,18 @@ opcache.memory_consumption=64
 opcache.max_accelerated_files=60000
 ```
 
+#### APCU
+
+We recommend enabling the [PHP APCu extension](https://getcomposer.org/doc/articles/autoloader-optimization.md#optimization-level-2-b-apcu-cache) and [configuring `composer` to support it]({{ page.baseurl }}/performance-best-practices/deployment-flow.html#preprocess-dependency-injection-instructions) to optimize for maximum performance. This extension caches file locations for opened files, which increases performance for Magento server calls including pages, ajax calls, and endpoints.
```
Comment (keharper) [inline src/guides/v2.3/performance-best-practices/software.md]:
```suggestion
We recommend enabling the [PHP APCu extension](https://getcomposer.org/doc/articles/autoloader-optimization.md#optimization-level-2-b-apcu-cache) and [configuring `composer` to support it]({{ page.baseurl }}/performance-best-practices/deployment-flow.html#preprocess-dependency-injection-instructions) to optimize for maximum performance. This extension caches file locations for opened files, which increases performance for Magento server calls including pages, Ajax calls, and endpoints.
```

File: src/guides/v2.2/performance-best-practices/software.md (hunk 2 above)
Comment (meker12) [reply src/guides/v2.2/performance-best-practices/software.md]: In the default .ini file, the acp.enabled is already there at the top, so it's just adding the extension.

File: src/guides/v2.2/performance-best-practices/software.md [hunk 4]
```diff
@@ -95,6 +95,18 @@ opcache.memory_consumption=64
 opcache.max_accelerated_files=60000
 ```
 
+#### APCU
+
+We recommend enabling the [PHP APCu extension](https://getcomposer.org/doc/articles/autoloader-optimization.md#optimization-level-2-b-apcu-cache) and [configuring `composer` to support it]({{ page.baseurl }}/performance-best-practices/deployment-flow.html#preprocess-dependency-injection-instructions) to optimize for maximum performance. This extension caches file locations for opened files, which increases performance for Magento server calls including pages, Ajax calls, and endpoints.
+
+Edit your `apcu.ini` file to include the following:
```
Comment (meker12) [inline src/guides/v2.2/performance-best-practices/software.md]:
```suggestion
Edit your `apcu.ini` configuration file to add the APCu extension module:
```

**Comment by meker12** [comment]
running tests

**Comment by meker12** [comment]
running tests

**Comment by meker12** [comment]
running tests

**Pull request merged**